### ISSUE TITLE
Suggested feature row: Image Tagging

### ISSUE THREAD

**Issue by Targren** [meichthys/foss_photo_libraries#41]
"Tags", "keywords", whatever you call them, they're pretty much my killer feature.

"Batch tagging" (applying the same tag to multiple images at once) would be even better, but that's probably too niche. :)

**Comment by meichthys** [comment]
Agreed. This should be added. I will try to get around to it, but a pull request would definitely be welcome!
